barretenberg/cpp/src/barretenberg/vm/tests/AvmMini_bitwise.test.cpp

```diff
@@ -188,57 +191,63 @@ TEST_F(AvmMiniBitwiseTestsU128, BitwiseNot)
  * See AvmMini_arithmetic.cpp for explanation of negative tests
  ******************************************************************************/
 
-std::vector<Row> gen_mutated_trace_not(FF const& a, FF const& c_mutated, avm_trace::AvmMemoryTag tag)
-{
-    auto trace_builder = avm_trace::AvmMiniTraceBuilder();
-    trace_builder.set(uint128_t{ a }, 0, tag);
-    trace_builder.bitwise_not(0, 1, tag);
-    trace_builder.halt();
-    auto trace = trace_builder.finalize();
-
-    auto select_row = [](Row r) { return r.avmMini_sel_op_not == FF(1); };
-    mutate_ic_in_trace(trace, select_row, c_mutated, true);
-
-    return trace;
-}
 /******************************************************************************
  * Negative Tests - FF
  ******************************************************************************/
 
-// This fails bitwise operations should not be used on FF-sized operands
-TEST_F(AvmMiniBitwiseNegativeTestsFF, BitwiseNot)
+TEST_F(AvmMiniBitwiseNegativeTestsFF, UndefinedOverFF)
 {
-    uint128_t input = 1;
-    std::vector<Row> trace = gen_mutated_trace_not(
-        FF{ uint256_t::from_uint128(input) }, FF{ uint256_t::from_uint128(~input) }, AvmMemoryTag::FF);
-    // EXPECT_EQ(alu_row.aluChip_alu_ff_tag, FF(1));
+    auto trace_builder = avm_trace::AvmMiniTraceBuilder();
+    // Triggers in write row 1 of mem_trace and alu_trace
+    trace_builder.set(10, 0, AvmMemoryTag::U8);
+    // Triggers a write in row 2 of alu_trace
+    trace_builder.op_not(0, 1, AvmMemoryTag::U8);
+    // Finally, we will have a write in row 3 of the mem_trace to copy the result
+    // from the op_not operation.
+    // Manually update the memory tags in the relevant trace;
+    auto trace = trace_builder.finalize();
+    for (size_t i = 1; i < 4; i++) {
```

Comment:
This will not bite us, because we only have the witness generation for the SET opcode and not the relations. Once we have them, this test will probably fail. It is ok for now, but can you add a short TODO: explaining this.?